README.md

```diff
@@ -36,7 +36,7 @@ First, add the dependency:
     <version>1.0.2</version>
 </dependency>
 ```
-Secondly, initialise the migrator by injecting the only dependency to it, Camundas ProcessEngine. For example:
+Secondly, initialise the migrator by injecting Camundas ProcessEngine and a MigrationInstructions object. This object be "empty" or null if you do not intend to specify migration instructions, which are only required for minor migrations. For example:
```

Comment:
It sounds like there is a word missing at: "...This object be "empty"  -> ... object **can** be "empty" ...

Reply:
:+1: